suite-native/navigation/src/navigators.ts

```diff
@@ -21,7 +22,10 @@ import {
 type ReceiveAccountsParams = {
     accountKey?: AccountKey;
     tokenContract?: TokenAddress;
-};
+} & RequireAllOrNone<
+    { networkSymbol?: NetworkSymbol; accountType?: AccountType; accountIndex?: number },
+    'networkSymbol' | 'accountType' | 'accountIndex'
+>;
```

Comment:
Please extract this to something like `AddCoinFlowParams` and reuse below to avoid duplicity

Reply:
ab1ff66d66